core/src/main/java/io/grpc/internal/AutoConfiguredLoadBalancerFactory.java

```diff
@@ -43,11 +47,19 @@
   static final String GRPCLB_LOAD_BALANCER_FACTORY_NAME =
       "io.grpc.grpclb.GrpclbLoadBalancerFactory";
 
-  AutoConfiguredLoadBalancerFactory() {}
+  @Nullable
+  private final ChannelTracer channelTracer;
+  @Nullable
+  private final TimeProvider timeProvider;
+
+  AutoConfiguredLoadBalancerFactory(ChannelTracer channelTracer, TimeProvider timeProvider) {
```

Comment:
nit: use `@Nullable` here as well to be consistent

Reply:
Done.